pages/learn/manage/account.md

```diff
@@ -38,17 +38,61 @@ __Warning__: This is your **only** opportunity to see the key, so make sure to d
 
 After you close the dialog, you'll see your API key in the list, complete with name, date of creation, and description:
 
-<img src="/img/common/preferences/api_key_list.png" width="80%">
+<img src="/img/common/preferences/api_key_list.png" width="100%">
 
 To revoke one or more API keys, select the boxes to the left of the tokens you wish to remove, then click *Delete selected*:
 
-<img src="/img/common/preferences/api_key_delete.png" width="80%">
+<img src="/img/common/preferences/api_key_delete.png" width="100%">
 
 API keys can also be generated using the API, [CLI][cli-keys], and [Node.js][node-sdk-keys] and [Python][python-sdk-keys] SDKs.
 
-## Collaboration management
+## Application members
 
-An organization should create a main account to host all applications that the organization owns. This allows a strict separation between applications the organization owns and employee applications created via their accounts. The main account is bound to the organization itself—the organization should have a well defined process to manage the credentials for its main account. Employees are granted access to the organization applications as collaborators. When an employee should no longer have access to the organization applications, access can be revoked by removing them as a collaborator.
+When an application needs to be shared with more than one person, the application owner can add new members. New application members can be given an appropriate level of access for their role, based on the following member types:
```

Comment:
Would it make sense to say "more than one users" rather than "more than one person"?